.gitignore

```diff
@@ -14,3 +14,18 @@ api-gen/build/
 
 # IDEs
 .vscode
+# Generated files by hugo
+/public/
+/resources/_gen/
+/assets/jsconfig.json
```

Comment:
Ditto - these are absolute paths. Please only use relative paths for the .gitignore file.